crates/router/src/core/payments/helpers.rs

```diff
@@ -6269,3 +6272,14 @@ pub fn validate_platform_fees_for_marketplace(
     }
     Ok(())
 }
+
+pub fn get_locale_from_header(headers: Option<&HeaderMap>) -> String {
```

Comment:
moved the `get_locale_from_header` to `crates/router/src/utils.rs` instead  and using the same fn now for payouts and payments, since the files where we are consuming this fn is not inside `payments` / `payouts` module